snap-example.cabal

```diff
@@ -2,10 +2,10 @@
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: c7f0e45cb6b400590c94547ad8f80bd079187895126043ce8f84f9a6b2af8a6a
+-- hash: 908cb6b71c9871d5ef7a55febb823b0c357cfdcb09d0178810b156e3b01596c9
 
 name:           snap-example
-version:        1.0.5
+version:        1.0.6
```

Comment:
About versioning, check these two articles:
https://semver.org/
https://blog.codinghorror.com/whats-in-a-version-number-anyway/

Reply:
Thanks @dbalseiro, gonna read those links right away.